### ISSUE TITLE
Add minor features to das-cli and update documentation

### ISSUE THREAD

**Issue by andre-senna** [singnet/das-toolbox#51]
- [x] `das-cli --version`
- [x] `das-cli update-version [--version]` (defaulted to newest version)
- [x] Rename `das-cli server` to `das-cli db`
- [x] `das-cli db restart`
- [x] `das-cli faas restart`
- [x] Remove parameter `--path` in `das-cli metta load` and `das-cli metta validate` and get the input file as a required parameter. The full command line should look like "das-cli metta load /opt/das/data/animals.metta`
- [x] Rename `das-cli metta validate` to `das-cli metta check`.
- [x] `das-cli server start` should sleep for a couple of seconds after finishing the startup of DB containers to make sure those containers are totally ready, otherwise if the user use this command in a script followed immediately by a `das-cli faas start`, this last command may fail because the DBs are not ready yet.
- [ ] Show progress bar printed by the metta parser binaries (db_loader and syntax_check) when executing `das-cli metta load` and `das-cli metta validate` (NOTE: db_loader and syntax_check don't print out progress bar for very small files)
- [x] Change message showing default version of the running function to show the actual version number. Something like "You are running the version 'latest' (1.21.12) of the function."
- [x] Add a `das-cli logs das` to follow the DAS log `das.log`
- [x] Add `das-cli jupyter-notebook start` start a jupyter-notebook server running with all required dependencies to use `hyperon-das`.
- [x] Adjust runtime messages for `das-cli example local` and `das-cli example faas`. Both show python commands
- [x] `das-cli faas update-version [--version]`
- [x] `das-cli faas --version`

**Comment by levisingularity** [comment]
@andre-senna Regarding the issue of the progress bar not being displayed, I conducted a test by accessing the container with the latest version of das-metta-parser (0.3.4) and then copied the animals.metta file into the container. I ran both the syntex_check and db_loader, but neither of them displayed the progress bar in the terminal. Below is a screenshot showing the output of both binaries respectively. It might be a problem with the loader.

![Image](https://github.com/singnet/das-toolbox/assets/141682181/2891a6bf-5c86-43c5-9697-052b41a40849)

![Image](https://github.com/singnet/das-toolbox/assets/141682181/17b16119-6317-48fc-8cac-0b9345fe07f1)
